digitizer/parse.py

```diff
@@ -67,8 +69,8 @@ def prepare_isotherm_dict(form):
         data['compositionType'] = form.inp_composition_type.value
         data['concentrationUnits'] = form.inp_concentration_units.value
     data['articleSource'] = form.inp_source_type.value
-    if 'table' in form.inp_source_type.value.lower():
-        data['tabular'] = True
+    if form.inp_tabular or 'table' in form.inp_source_type.value.lower():
```

Comment:
The string-based check was put in place before the checkbox was added.
I think it should be deleted now (we should not consider data as tabular if the checkbox is not checked)

```suggestion
    if form.inp_tabular.value:
```

Reply:
Agree, I was debating this back and forth. Setting data to tabular=True should be an active, affirmed decision. I will add some text instructions to the form as well.